apis/config/v1beta1/configuration_types.go

```diff
@@ -318,9 +318,15 @@ const (
 )
 
 type InternalCertManagement struct {
-	// Enable controls whether to enable internal cert management or not.
-	// Defaults to true. If you want to use a third-party management, e.g. cert-manager,
-	// set it to false. See the user guide for more information.
+	// Enable controls the use of internal cert management for the webhook
+	// and metrics endpoints.
+	// When enabled Kueue is using libraries to generate and
+	// self-sign the certificates.
+	// When disabled, you need to provide the certificates for
+	// the webhooks and metrics through a third party certificate
+	// This secret is mounted to the kueue controller manager pod. The mount
+	// path for webhooks is /tmp/k8s-webhook-server/serving-certs, whereas for
+	// metrics endpoint the expected path is `/etc/kueue/metrics-certs`.
```

Comment:
Could we mention fixed name certification names, `tls.crt` and `tls.key`?

Reply:
fixed.